docs/auth/phone-auth.md

```diff
@@ -84,3 +84,60 @@ Enter a new phone number (e.g. `+44 7444 555666`) and a test code (e.g. `123456`
 
 Once added, the number can be used with the `signInWithPhoneNumber` method, and entering the code specified will
 cause a successful sign-in.
+
+
+# MFA Authentication
+
```

Comment:
Could you add something here that indicates firebase is not a standalone component in an MFA system? It appears to require enrollment in the Google Cloud Identify Platform as well? Anything similar to the answer here might serve well: https://stackoverflow.com/a/60719962/9910298

Reply:
How is this?